Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4-210
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4-212
```

Comment:
A new 8.5-204 version got released ~10 hours ago, we could try that too
https://catalog.redhat.com/software/containers/ubi8/ubi-minimal/5c359a62bed8bd75a2c3fba8?tag=8.5-204&push_date=1636504224000

Reply:
resolved sorry for the delay